test/template.test.js

```diff
@@ -169,5 +180,35 @@ describe('JsonTemplate', function() {
 
       done();
     });
+
+    it('should ignore invalid vars', function(done) {
+      var obj = {
+        url: 'http://localhost:3000/test',
+        headers: {
+          'x-lookup-request': '{x+y}{|}{0a}{=4}',
+        },
+      };
+      var template = new JsonTemplate(obj);
+      var result = template.build(template);
+      result.should.be.eql(obj);
```

Comment:
Sorry @raymondfeng, can you elaborate how this is testing that the invalid var is ignored?

Reply:
Since they are not recognized as variables and not substituted by `build`, the result will be the same as the template.

Reply:
For some reason, I can't use reactions, but 👍